air_quality.cpp

```diff
@@ -140,8 +140,7 @@ void DeRestPluginPrivate::handleAirQualityClusterIndication(const deCONZ::ApsDat
 
         if (configUpdated || stateUpdated)
         {
-            updateEtag(sensor->etag);
-            updateEtag(gwConfigEtag);
```

Comment:
Note the `gwConfigEtag` needs to be updated as well since querying the full config state is affected by this.

Reply:
Using `updateSensorEtag(&*sensor);` instead does take care of that, doesn't it?